library/src/main/java/com/q42/q42stats/library/HttpService.kt

```diff
@@ -0,0 +1,48 @@
+package com.q42.q42stats.library
+
+import org.json.JSONException
+import org.json.JSONObject
+import java.io.BufferedWriter
+import java.io.IOException
+import java.io.OutputStreamWriter
+import java.net.HttpURLConnection
+import java.net.URL
+import javax.net.ssl.HttpsURLConnection
+
+object HttpService {
+    /** Synchronously send the stats. Make sure to run this on a worker thread */
+    fun sendStatsSync(config: Q42StatsConfig, data: JSONObject) {
+        sendStatsSync(
+            "https://firestore.googleapis.com/v1/projects/${config.fireBaseProject}/" +
+                    "databases/(default)/documents/${config.firebaseCollection}?mask.fieldPaths=_",
+            data
+        )
+    }
+
+    /** Synchronously send the stats. Make sure to run this on a worker thread */
+    private fun sendStatsSync(url: String, data: JSONObject) {
+        httpPost(url, data)
+    }
+
+    @Throws(IOException::class, JSONException::class)
+    private fun httpPost(url: String, jsonObject: JSONObject) {
+        val conn = URL(url).openConnection() as HttpsURLConnection
```

Comment:
Maybe next time use okhttp (HttpsURLConnection is a wrapper around okhttp)

Reply:
I prefer not to depend on any external libraries, for file size and gradle version conflict reasons

```
import java.net.HttpURLConnection
import java.net.URL
import javax.net.ssl.HttpsURLConnection
```

Reply:
👍  underneath it uses okhttp (okhttp is part of Android), but I see that using the okhttp API needs an extra library. Then your solution is better.